.circleci/scripts/pre_commit_readme_extra.py

```diff
@@ -10,7 +10,7 @@
 config.read(repo_dir / "setup.cfg")
 
 all_extra = []
-extra_to_exclude = {"tests", "mypy", "docs"}
+extra_to_exclude = {"tests", "mypy", "docs", "test_python_3_11"}
```

Comment:
shall we add this in README or not?

Reply:
I think we should exclude this as it's only for testing purposes.